spinners.json

```diff
@@ -867,18 +867,34 @@
 	"christmas": {
 		"interval": 400,
 		"frames": [
-			"🌲", 
+			"🌲",
 			"🎄"
 		]
 	},
+	"grenade": {
+		"interval": 80,
+		"frames": [
+			"،   ",
+			"′   ",
+			" ´ ",
+ 			" ‾ ",
+			"  ⸌",
+			"  ⸊",
+			"  |",
+			"  ⁎",
+			"  ⁕",
+			" ෴ ",
+			"  ⁓"
```

Comment:
I think we should add maybe 3 empty frames here. This animation needs a short pause before replaying.

```json
			"   ",
			"   ",
			"   "
```

Reply:
yeah, looks better now.